packages/package-json-lint/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@cerner/package-json-lint",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Package JSON linter",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git",
+    "directory": "packages/package-json-lint"
+  },
+  "keywords": [
+    "lint",
+    "package.json",
```

Comment:
I'm not sure package.json is needed s a keyword

Reply:
I'm fine removing this.